readme.md

```diff
@@ -48,6 +100,19 @@ Default: `name` field in your package.json
 
 The name used to identify it.
 
+##### version
+
+Type: `string`<br>
+Default: `undefined`.
+
+The version used to identify it.
+
+##### detectVersion
+
+Type: `boolean`<br>
+Default: `false`.
+
+If set to `true` and `name` detected from `package.json`, `version` will be also detected.
```

Comment:
This does not make it clear that `firstRun()` will return true for each new version.